core/templates/pages/contributor-dashboard-page/services/translate-text.service.ts

```diff
@@ -146,17 +150,17 @@ export class TranslateTextService {
       contentType: contentType,
       interactionId: interactionId,
       ruleType: ruleType
-    };
+    }as TranslatableItem;
   }
 
   init(expId: string, languageCode: string, successCallback: () => void): void {
     this.stateWiseContentIds = {};
     this.stateNamesList = [];
     this.stateAndContent = [];
     this.activeIndex = this.STARTING_INDEX;
-    this.activeContentId = null;
-    this.activeStateName = null;
-    this.activeContentText = null;
+    this.activeContentId;
+    this.activeStateName;
+    this.activeContentText;
```

Comment:
`init` function is for initializing these values, why are we removing the assignments here ?

Reply:
They all are defined as types other than null, due to which assignment of null value to them is throwing error. So I removed the assignment.
Can we remove this error in some other way?

Reply:
As they are defines as some other types in lines 76,77,78. Initializing them with null is throwing an error.

Reply:
Hmm, I'm not exactly sure, leaving them undefined here means that they shouldn't present here at all. You'll need to check the behaviour of how it affects the usage of this property. These changes were introduced with this PR https://github.com/oppia/oppia/pull/12727.